docs/mcp/configuring-mcp-servers.mdx

```diff
@@ -101,17 +115,17 @@ STDIO configuration example:
 
 ```json
 {
-	"mcpServers": {
-		"local-server": {
-			"command": "node",
-			"args": ["/path/to/server.js"],
-			"env": {
-				"API_KEY": "your_api_key"
-			},
-			"alwaysAllow": ["tool1", "tool2"],
-			"disabled": false
-		}
-	}
+  "mcpServers": {
+    "local-server": {
+      "command": "node",
+      "args": ["/path/to/server.js"],
+      "env": {
+        "API_KEY": "your_api_key"
+      },
+      "alwaysAllow": ["tool1", "tool2"],
+      "disabled": false
+    }
+  }
```

Comment:
Note I haven't changed the STDIO or SSE examples other than the auto-formatter setting the indentation to spaces as the Cline docs seem to have a mix of tabs and spaces. Happy to leave these alone if you'd prefer.